spec/features/mi_bridges_driving_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 RSpec.feature "MI Bridges Driving" do
   # to run this in a non-headless way, remove the `:js` flag below
-  scenario "successfully drives a SNAP Application", :js, :driving do
+  scenario "successfully drives, re-drives a SNAP Application", :js, :driving do
```

Comment:
`successfully drives, re-drives a SNAP Application`

=> `successfully drives, then re-drives, a SNAP Application`

Something about that initial wording felt a little awkward.

Reply:
good call, update at 42ef937